Villager Trades mod element (#1557)
MCreator/MCreator: plugins/mcreator-localization/help/default/villagertrades/price2.md

The second item the trade requires.

> NOTE: Optional

Reviewer: Второй предмет, необходимый (если указан) для заключения сделки.

Reviewer: I'd also shorten the original tip since making a NOTE for a single word is quite strange.
```suggestion
The second item required by the trade (if selected).
```

Author: No, second item is not required, why "if selected"?

Reviewer 2: > No, second item is not required, why "if selected"?

It is required if the second item is selected
